client/src/components/teams/CreateTeam.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from "react";
+import {
+  Dialog,
+  makeStyles,
+  Button,
+  DialogContent,
+  DialogTitle,
+  DialogActions,
+  InputLabel,
+  TextField,
+  Grid,
+} from "@material-ui/core";
+
+import InviteMembers from "./InviteMembers";
+import MembersToBeInvited from "./MembersToBeInvited";
+
+const CreateTeam = ({ open, closeDialog }) => {
+  const classes = useStyles();
+
+  const createTeam = (event) => {
+    event.preventDefault();
+  };
+
+  const [teamName, setTeamName] = useState("");
+  const handleTeamNameChange = (event) => {
+    event.preventDefault();
```

Comment:
`preventDefault` is not necessary in these cases of typing in a textfield. The main case for when you need it is when you have a `type="submit"` button